src/user/layouts/signup/SignUp.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react'
+import SignUpFormContainer from '../../ui/signupform/SignUpFormContainer'
+
+class SignUp extends Component {
+  render() {
+    return(
+      <main className="container">
+        <div className="pure-g">
+          <div className="pure-u-1-1">
+            <h1>Sign Up</h1>
+            <p>We've got your wallet information, simply input your name and your account is made!</p>
```

Comment:
Escape '?

Reply:
This place is HTML, just like plain English...  ' is not for string.